src/app/personside/infotabs/saksoversikt/DokumentKomponent.tsx

```diff
@@ -130,56 +171,75 @@ class DokumentKomponent extends React.Component<Props> {
         const saksid = dokument.tilhørendeFagsaksid ? dokument.tilhørendeFagsaksid : dokument.tilhørendeSaksid;
         const saksvisning = this.props.sakstemakode === sakstemakodeAlle ?
             (
-                <Normaltekst>Saksid: {saksid} / Sakstema: {this.props.sakstemanavn}</Normaltekst>
+                <EtikettGrå>Saksid: {saksid} / Sakstema: {this.props.sakstemanavn}</EtikettGrå>
             ) :
             (
-                <Normaltekst>Saksid: {saksid}</Normaltekst>
+                <EtikettGrå>Saksid: {saksid}</EtikettGrå>
             );
 
         const vedlegg = dokument.vedlegg && dokument.vedlegg.length > 0 ?
             (
-                <VedleggStyle>
+                <div>
                     <Normaltekst>Dokumentet har {dokument.vedlegg.length} vedlegg:</Normaltekst>
                     <ul>
                         {dokument.vedlegg.map(vlegg =>
                             <li key={vlegg.dokumentreferanse + dokument.journalpostId}>
-                                <a
-                                    href={'#'}
-                                    onClick={() => this.visDokumentHvisTilgang(dokument, vlegg)}
-                                    ref={this.vedleggLinkRef}
-                                >
-                                    {vlegg.tittel}
-                                </a>
+                                <span ref={this.vedleggLinkRef}>
+                                    <DokumentLenkeStyle
```

Comment:
Har du prøvd å bruke denne med skjermleser? Knapper med mye innhold kan være litt vanskelig for skjermlesere